external/ocsp/ocsp/agent.js

```diff
@@ -67,6 +67,10 @@ Agent.prototype.createConnection = function createConnection(port,
 Agent.prototype.handleOCSPResponse = function handleOCSPResponse(socket,
                                                                  stapling,
                                                                  cb) {
+  if (process.env.NODE_TLS_REJECT_UNAUTHORIZED === '0') {
+    cb();
+    return;
```

Comment:
This should likely go in WebSocketMessageAdapter to not register the agent in the first place if checks are going to be skipped.

There's two reasons for this:
1) I'd like to keep as close to the ocsp library as possible in the hopes of getting the changes made so far committed back to the origin.

2) This doesn't cover the scenario where a proxy is used.

Reply:
Ok, I pushed a revised commit. Is it better?